docs/development/plugins/data/public/kibana-plugin-plugins-data-public.baseformatterspublic.md

```diff
@@ -7,5 +7,5 @@
 <b>Signature:</b>
 
 ```typescript
-baseFormattersPublic: (import("../../common").FieldFormatInstanceType | typeof DateFormat)[]
+baseFormattersPublic: (typeof DateFormat | import("../../common").FieldFormatInstanceType)[]
```

Comment:
I have no clue why order of parameters changed in the docs
There are no API changes in this PR :-(